src/frontend/qt_sdl/Input.cpp

```diff
@@ -28,6 +28,7 @@ namespace Input
 
 int JoystickID;
 SDL_Joystick* Joystick = nullptr;
+SDL_Haptic *Haptic = nullptr;
```

Comment:
Could you change this to keep with the current coding style?
```suggestion
SDL_Haptic* Haptic = nullptr;
```